library/commandline/test/clients/dummy_cmdline.rb

```diff
@@ -2,6 +2,7 @@ module Yast
   class DummyCmdlineClient < Client
     def main
       Yast.import "CommandLine"
+      textdomain "base"
```

Comment:
Please use the "example" text domain, otherwise all texts from this file will end up in the `base.pot` file and the translators will translate them. The `example.pot` file is ignored (not sent to weblate).

```suggestion
      textdomain "example"
```